### DISCUSSION TITLE
Dim the deselected panes

### DISCUSSION THREAD

**Discussion by joexue** [wezterm/wezterm#1382] [Ideas]
With multiple panes in one tab/window, dim the deselected panes make the selected one outstanding.

**Comment by permelin** [comment]
See this config:
https://wezfurlong.org/wezterm/config/appearance.html?highlight=inactive_pane_hsb#styling-inactive-panes

**Reply by joexue** [reply]
exactly what I need, did not find it before, thank you very much

**Reply by permelin** [reply]
I've been digging through the docs for hours and I'm still finding new things.
